src/components/AddStudent.js

```diff
@@ -0,0 +1,67 @@
+import * as React from "react";
+import Button from '@mui/material/Button';
+import TextField from '@mui/material/TextField';
+ 
+class AddStudent extends React.Component {
+  constructor(props) {
+    super(props);
+    //Neccessary information to post and communicate errors/success
+    this.state={ name:'',uEmail:'', message:'' };
+  }
+  
+  
+  handleSubmit = () => {
+    //Do not accept incomplete data
+    if ( this.state.name == '' || this.state.uEmail == '' )  {  
+        this.setState({message: 'Enter complete info'});
+        return;
+    }
+    //Call the POST to backend
+    fetch('http://localhost:8080/student/add',
+    {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({
+        name: this.state.name,
+        email: this.state.uEmail,     
+      })
+    }) //Now update message for the response from backend
+      .then(response => response.json() )
+      .then(responseData => {
+        const { message } = responseData;
+        this.setState({ 
+          message: message 
```

Comment:
If your variable name is the same as your key, you can always use the shorthand.

```
this.setState({
  message,
})
```

Reply:
Got it.